dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -721,7 +721,13 @@ public void sortChildren(String inode)  throws DotDataException {
 			statement = conn.createStatement();
 			String sql;
 
-            PreparedStatement createSortPreparedStatement = conn.prepareStatement( catSQL.getCreateSortChildren() );
+            if ( DbConnectionFactory.isOracle() ){
+                //For Oracle we need to avoid ORA-01027 by creating the table before.
+                sql = catSQL.createCategoryReorderTable();
+                statement.execute( sql );

```

Comment:
Autocommit is set to false, the thing is that the table created by the method is used like a DB view and we don't need it outside the current logic, so IMO we shouldn't commit at that point because an error could leave us in a bad state. 

Do you have a case in mind where the autocommit false could affect us?